scripts/linters/excluded_any_type_files.json

```diff
@@ -9,6 +9,11 @@
     "core/templates/domain/classifier/AnswerClassificationResultObjectFactory.ts",
     "core/templates/domain/classifier/AnswerClassificationResultObjectFactorySpec.ts",
     "core/templates/domain/classroom/classroom-backend-api.service.ts",
+<<<<<<< HEAD
+    "core/templates/domain/collection/CollectionNodeObjectFactory.ts",
+    "core/templates/domain/collection/CollectionObjectFactory.ts",
+=======
+>>>>>>> develop
```

Comment:
I think you didn't resolve the conflicts properly?

Reply:
done